components/core/src/os/process/windows.rs

```diff
@@ -32,6 +32,16 @@ pub fn become_command(command: PathBuf, args: Vec<OsString>) -> Result<()> {
     become_child_command(command, args)
 }
 
+/// Get process identifier of calling process.
+pub fn current_pid() -> u32 {
+    unsafe { kernel32::GetCurrentProcessId() as u32 }
+}
+
+/// Determines if a process is running with the given process identifier.
+pub fn is_alive(pid: u32) -> bool {
+    handle_from_pid(pid).is_some()
```

Comment:
@mwrock yeah I just took a peek at the docs and saw this and updated the PR - great catch! I made the changes in a separate commit to show the cleanup more clearly